README.md

```diff
@@ -10,7 +10,8 @@ it does use npm to manage its dependencies
 ## Build
 
 ```sh
-$ npm install -g iced-coffee-script
+$ sudo ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
Can you remove these changes?  I'm assuming you've installed node as your user and it's in your path.

Reply:
Ok, I will remove this.